### ISSUE TITLE
[Feature Request] Disable per-URL

### ISSUE THREAD

**Issue by yllekz** [ACK-J/Port_Authority#10]
**Is your feature request related to a problem? Please describe.**
I have a local NAS serving up a noVNC instance. This extension gets cranky about it and prevents me from loading the page. I have to either disable the extension or reload the page. It would be great if the extension had a "disable for this site" option or at least a quick way to manage exceptions.

**Comment by ACK-J** [comment]
Yea I agree this needs to be added. I'll try to get to it tomorrow unless you are willing to submit a PR

**Comment by yllekz** [comment]
Hello, I'm not sure how to do that honestly.

**Comment by frjoethesecond** [comment]
Hi there. Any updates on the whitelist? The addon is great, but it breaks a lot of local stuff like router interfaces, and AI frontends.

**Comment by psiberfunk** [comment]
@ACK-J I agree that this breaks a lot of localhost /local LAN stuff people may need.  Whitelist would be ideal

**Comment by ACK-J** [comment]
@psiberfunk @frjoethesecond @yllekz This is actively being worked on by @jjk1492 and many of the changes are already in the master branch. Expect a big update in the next week or so.

**Comment by jjk1492** [comment]
@psiberfunk @frjoethesecond @yllekz As of yesterday whitelisted has been merged in, @ACK-J not sure what the release process is like

**Comment by frjoethesecond** [comment]
Thanks for your hard work. It's working perfectly now.

**Comment by ACK-J** [comment]
The domain whitelist feature is now live in v2.0.0 https://github.com/ACK-J/Port_Authority/releases/tag/2.0.0
Your browser should automatically update the extension within a day or so
